prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchema.kt

```diff
@@ -158,6 +128,130 @@ data class ConfigSchema(
         check(elementsInSchema.size <= 1)
         return if (elementsInSchema.isEmpty()) null else elementsInSchema[0]
     }
+
+    override fun equals(other: Any?): Boolean {
+        return other != null &&
+            other is ConfigSchema<*> &&
+            elements == other.elements &&
+            constants == other.constants &&
+            extends == other.extends
+    }
+
+    override fun hashCode(): Int {
+        return elements.hashCode() * 31 + constants.hashCode() * 30 + extends.hashCode() * 29
+    }
+}
+
+/**
+ * A converter schema.
+ * @property name the schema name
+ * @property hl7Type the HL7 message type for the output.  Only allowed at the top level schema
+ * @property hl7Version the HL7 message version for the output.  Only allowed at the top level schema
+ * @property elements the elements for the schema
+ * @property constants schema level constants
+ * @property extends the name of a schema that this schema extends
+ */
+@JsonIgnoreProperties
+class ConverterSchema(
```

Comment:
Separate these into their own files. We want to keep all that logic very separate. They will be in separate folders with any and all code specific to each type of converter.  Also means the common code needs to be in some common folder.

Reply:
Separated out in latest commit